src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/locations/LocationsDomainEventListener.kt

```diff
@@ -37,8 +37,6 @@ class LocationsDomainEventListener(
       "location.inside.prison.amended" -> locationsService.amendLocation(message.fromJson())
       "location.inside.prison.deactivated" -> locationsService.deactivateLocation(message.fromJson())
       "location.inside.prison.reactivated" -> locationsService.reactivateLocation(message.fromJson())
-      "location.inside.prison.capacity.changed" -> locationsService.changeCapacity(message.fromJson())
-      "location.inside.prison.certification.changed" -> locationsService.changeCertification(message.fromJson())
```

Comment:
these capacity and certification messages were not implemented in the end - they are done as part of the amend event